internal/cmprefimpl/cmppg/checks.go

```diff
@@ -260,19 +260,15 @@ func checkDimension(t *testing.T, want UnaryResult, g geom.Geometry) {
 
 func checkEnvelope(t *testing.T, want UnaryResult, g geom.Geometry) {
 	t.Run("CheckEnvelope", func(t *testing.T) {
-		if g.IsEmpty() {
-			// PostGIS allows envelopes on empty geometries, but they are empty
-			// envelopes. In simplefeatures, an envelope is never empty, so we
-			// skip testing that case.
+		got := g.Envelope().AsGeometry()
+		want := want.Envelope
+
+		// The geometry type for empty envelopers is different for
```

Comment:
Not sure if this is what you meant.
```suggestion
		// The geometry type for empty envelopes is different for
```

Reply:
Ahh, yes that's what I meant. Fixed in 1e47e80742c5cce93476196eaa3a0be051c1c3de